website/docs/migration-guides/yew/0.18.0-to-0.19.0.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "0.18.0 to 0.19.0"
+---
+
+`Yew 0.19.0` has changed a lot, thus this migration will not cover ALL of the changes.
+
+Instead only the most impacting changes are mentioned and the rest should be picked up by cargo.
+
+## html! requirement for braces around most props
+
+Put it simply almost all the time you have to provide braces for props:
+
+```rust {4}, ignore
+let super_age = 1;
+html!{
+    <JapaneseYew
+        age=super_age // ! Will throw an error
+    >
+}
+```
+
+```rust {4}, ignore
+let super_age = 1;
+html!{
+    <JapaneseYew
+        age={super_age} // Correct
+    >
+}
+```
+
+Also now you can use a shorthand if prop and variable names are the same:
+
+```rust {4}, ignore
+let age = 1;
+html!{
+    <JapaneseYew
+        {age}
+    >
+}
+```
+
+There is a community provided regex to help with this change, though we cant promise it will work all the time.
+
+find with `=(?![{">=\s])([^\s></]*(\s!{0,1}[=|&]{2}\s[^\s></]*)*)`
```

Comment:
It breaks if there's a closure in the mix. I used it while working on the benchmark thing and it broke when it encountered `|_|`